helmfile.d/nginx-ingress.yaml

```
@@ -37,6 +37,7 @@ releases:
             # I doubt we need it at the moment, hence this comment.
             # https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#use-geoip2
             use-geoip2: "true"
+            hsts-max-age: "2592000" # 30 days - INFRA-2998
```

no reason to use the minimum?

```suggestion
            hsts-max-age: "31536000" # 1 year - INFRA-2998
```

unless you just want to test it first?

---

Yes, I wondered, but it seems (as per https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#configuration-options)  that `15724800` is the default (e.g. 182 days)